generators/cypress/templates/src/test/javascript/cypress/e2e/account/settings-page.cy.ts.ejs

```diff
@@ -22,22 +22,48 @@ import {
   submitSettingsSelector,
   emailSettingsSelector,
 } from '../../support/commands';
+import type { Account } from '../../support/account';
 
 describe('/account/settings', () => {
   const adminUsername = Cypress.env('E2E_USERNAME') ?? 'admin';
   const adminPassword = Cypress.env('E2E_PASSWORD') ?? 'admin';
   const username = Cypress.env('E2E_USERNAME') ?? 'user';
   const password = Cypress.env('E2E_PASSWORD') ?? 'user';
 
-  beforeEach(() => {
+  const testUserEmail = 'user@localhost.fr';
+  let originalUserAccount: Account;
+  let testUserAccount: Account;
+
+  before(() => {
     cy.login(username, password);
-    cy.visit('/account/settings');
+
+    cy.getAccount().then(account => {
+      originalUserAccount = account;
+      testUserAccount = { ...account, email: testUserEmail };
+
+      // need to modify email because default email does not match regex in some frameworks
```

Comment:
Why can't we just fix the regex in some frameworks?

Reply:
`localhost` is not a valid domain for emails.
Not sure which approach to use.

Reply:
We could use `example.com`, since that's what it's designed for. https://example.com/

Or, any of the jhipster domains. For example, I own jhipster.org and just redirect it to jhipster.tech.

Reply:
Looks good to use example.com.
I will create a issue about this.